EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -1131,14 +1131,14 @@ QUEST_LV_0200_20150317_001130	I used to travel in and out of town, but the pries
 QUEST_LV_0200_20150317_001131	I have to have a livelihood. {nl}The other villagers here just spend all day looking up to the priests and the goddess.
 QUEST_LV_0200_20150317_001132	I've seen other Revelators here that got all weird, but you seem fine.{nl}Did you meet the goddess or something? {nl}Well, you did some great work. I wish I was as strong as you are.
 QUEST_LV_0200_20150317_001133	Pharmacist Tiana
-QUEST_LV_0200_20150317_001134	Priest Dazine asked me for medicine to cure future addled Revelators with. {nl}But I ran out of Spring Light Grass.. and I'm scared of the demons.
+QUEST_LV_0200_20150317_001134	Priest Dazine asked me for medicine to cure future addled Revelators with. {nl}But I ran out of Spring Light Grass... and I'm scared of the demons.
```

Comment:
`to cure the future addled Relevators with, {nl} but I`
and maybe change `and I'm scared of the demons.`
to
`I'm also scared of the demons.` ?